README.md

```diff
@@ -46,35 +46,96 @@ Other devices may be compatible and all that is needed is to add the ID to apex-
 
 ## Installation
 
+For installing this software, follow these steps:
+
 ### UDev
 
-Enter the following data from [here](https://gist.github.com/ToadKing/d26f8f046a3b707e9e4b9821be5c9efc) (Shoutout to @ToadKing](https://github.com/ToadKing).
+1. Get the device id: `lsusb | grep "SteelSeries Apex"`:
+
+```shell
+$ lsusb | grep "SteelSeries Apex"
+Bus 001 Device 002: ID 1038:1610 SteelSeries ApS SteelSeries Apex Pro
+```
+
+The **id** is the right part of the ID.
 
-If those don't work and lead to an "Access denied" error please try the rules by [FrankGrimm](https://github.com/FrankGrimm) from [here](https://github.com/FrankGrimm/apex7tkl_linux).
+2. Enter the following data from [here](https://gist.github.com/ToadKing/d26f8f046a3b707e9e4b9821be5c9efc) (Shoutout [to @ToadKing](https://github.com/ToadKing)).
+
+If those don't work and lead to an "Access denied" error please try the following rules and save the rules as `97-steelseries.rules`:
+
+```shell
+cat /etc/udev/rules.d/97-steelseries.rules
+SUBSYSTEM=="input", GROUP="input", MODE="0666"
+
+SUBSYSTEM=="usb", ATTRS{idProduct}=="<PRODUCT ID HERE>", ATTRS{idProduct}=="1610", MODE="0666", GROUP="plugdev"
+KERNEL=="hidraw*", ATTRS{idProduct}=="<PRODUCT ID HERE >", ATTRS{idProduct}=="1610", MODE="0666", GROUP="plugdev"
+```
+
+1. Replace the `ATTRS{idProduct}==` value with the device **id**.
+
+2. Save all files to `/etc/udev/rules.d/97-steelseries.rules`.
+
+3. Finally, reload the `udev` rules: `sudo udevadm control --reload && sudo udevadm trigger`
 
 ### Rust
 
 - Install Rust **nightly** using [rustup](https://rustup.rs/)
 - Install required dependencies
   - For Ubuntu: `sudo apt install libssl-dev dbus-dev libusb-1.0-0-dev`
 - Clone the repository: `git clone git@github.com:not-jan/apex-tux.git`
-- Change directory into the repository: `cd apex-tux`
+- Change the directory into the repository: `cd apex-tux`
 - Compile the app using the features you want
   - If you **don't** run DBus you have to disable the dbus feature: `cargo build --release --no-default-features --features crypto,usb`
-  - Otherwise just run `cargo build --release`
+  - Otherwise, just run `cargo build --release --features sysinfo,hotkeys,gif`
```

Comment:
gif has now been replaced with image (still support gif, but add support for PNg, JPEG etc.)